recipes/realgud-debugger2

```diff
@@ -0,0 +1,3 @@
+(realgud-debugger2 :fetcher github :repo "rocky/realgud-ruby-debugger2"
```

Comment:
Here the `realgud-debugger2` name doesn't match the name of `realgud-rdb2.el` -- the package name and main contained filename should match, so you probably want to rename this package file.

Inside `realgud-rdb2.el` has `(defgroup realgud ...)` in it, which should presumably be `(defgroup realgud-rdb2 ...)`